pupa/cli/commands/update.py

```diff
@@ -59,6 +59,46 @@ def save_report(report, jurisdiction):
             )
 
 
+@transaction.atomic
+def save_report(report, jurisdiction):
+    from pupa.models import RunPlan
+    plan = RunPlan.objects.filter(jurisdiction=jurisdiction).latest('id')
```

Comment:
I thought about this, but I feel like we need to decouple the `import` and `report` phases. The `RunPlan` notes the status of the `import`, and the `Measures` is separately for `report`.